cmd/operator/deploy/operator/operator.yaml

```diff
@@ -81,7 +81,7 @@ spec:
       serviceAccountName: operator
       containers:
       - name: operator
-        image: gke.gcr.io/prometheus-engine/operator:v0.1.0-gke.0
+        image: gke.gcr.io/prometheus-engine/operator:v0.1.1-gke.0
```

Comment:
Can we mimic the args from examples/operator.yaml here?

Reply:
Done. Also mirrored the other way by adding priority class to the examples one.
We should get rid of the duped YAMLs soon.